tests/unittests/tests-hashes/tests-hashes-sha384-hmac.c

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright (C) 2016 Martin Landsmann <martin.landsmann@haw-hamburg.de>
+ * Copyright 2016 OTA keys S.A.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#include <limits.h>
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "psa/crypto.h"
+
+#include "embUnit/embUnit.h"
+
+#include "hashes/sha384.h"
+
+#include "tests-hashes.h"
+#include <psa_crypto/psa/crypto_types.h>
+#include <psa_crypto/psa/crypto_struct.h>
+
+static int compare_str_vs_digest(const char *str,
+                                 const uint8_t hash[SHA384_DIGEST_LENGTH])
+{
+    char ch[3] = {0, 0, 0};
+    size_t iter_hash = 0;
+    size_t str_length = strlen((char *)str);
+    for (size_t i = 0; i < str_length; i += 2)
+    {
```

Comment:
```suggestion
    for (size_t i = 0; i < str_length; i += 2) {
```

and in other places too (except for function definitions): https://github.com/RIOT-OS/RIOT/blob/master/CODING_CONVENTIONS.md#indentation-and-braces